client/frontend-trello/src/pages/workspace/board/AddBoard.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import {IBoard, IBoardModification} from "../../../models/dto/Board";
+import {AddBoardProp} from "./props";
+import Button from "@mui/material/Button";
+import {Card, CardContent, TextField, Typography} from "@mui/material";
+import "../../../styles/AddBoard.css";
+
+const AddBoard = ({onCreate}: AddBoardProp) => {
+
+    const [post, setPost] = useState<IBoardModification>({name: '', description: ''})
+
+    const addNewBoard = () => {
+        const newBoard: IBoard = {
+            id: Date.now(),
+            ...post,
+            createdAt: new Date(),
+            updatedAt: new Date()
+        }
+        onCreate(newBoard);
+        setPost({name: '', description: ''});
+    }
+
+    return (
+        <Card variant="outlined">
+            <CardContent>
+                <Typography className={'typography'} variant="h4">New Board</Typography>
+                <TextField
+                    variant="outlined"
+                    label="Name of board"
+                    value={post.name}
+                    onChange={(event) => setPost({...post, name: event.target.value})}
+                    type="text"
+                    fullWidth
+                    margin="normal"
+                />
+                <TextField
+                    variant="outlined"
+                    label="Description of board"
+                    value={post.description}
+                    onChange={(event) => setPost({...post, description: event.target.value})}
```

Comment:
You can destructure it as well, if you wish
```suggestion
                    onChange={({target: {value}}) => setPost({...post, description: value})}
```